### PR TITLE
fix(insured-bridge-relayer): Key deployTimestamps by lowercase addresses

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**

`deposit.l1Token` has checksummed address, while `RelayerConfig.deployTimestamps` keys by lower-case.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [X]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.ts [hunk 1]
```diff
@@ -980,7 +980,7 @@ export const defaultConfigs: { [name: string]: { type: string; [key: string]: an
     type: "insuredbridge",
     l2BlockLookback: 99000, // Arbitrum only supports lookbacks of 100,000 blocks
     l2NetId: 42161,
-    bridgeAdminAddress: "0xde7130ceaf4345a12a2764f0dc50bc4899d7ec08",
+    bridgeAdminAddress: "0x30B44C676A05F1264d1dE9cC31dB5F2A945186b6",
```
Comment (nicholaspai) [inline packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.ts:983]: Unrelated change to this file that we should make now that we've ugpraded BridgeAdmin

**Review by chrismaree — APPROVED** [review]
I modified this to use `replaceAddressCase` to not place restrictions on how the user configures the bot. In all cases, internally the bot should use check sum formatted addresses.

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.ts (hunk 1 above)
Comment (chrismaree) [reply packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.ts:983]: yup, this is clutch.
